pyvista/core/filters/data_set.py

```diff
@@ -1339,6 +1348,11 @@ def extract_all_edges(self, progress_bar=False):
         """
         alg = _vtk.vtkExtractEdges()
         alg.SetInputDataObject(self)
+        if use_all_points:
+            try:
+                alg.SetUseAllPoints(use_all_points)
+            except AttributeError:  # pragma: no cover
+                raise VTKVersionError('This version of VTK does not support `use_all_points=True`.')
```

Comment:
Can we document which version this feature was introduced?

Reply:
From test, it looks like 9.1

Reply:
```suggestion
            except AttributeError:  # pragma: no cover
                raise VTKVersionError(
                    'This version of VTK does not support `use_all_points=True`. '
                    'VTK v9.1 or newer is required.'
                )
```